cellxgene_schema_cli/cellxgene_schema/ontology.py

```diff
@@ -1,86 +1,201 @@
 """Methods for working with ontologies and the OLS."""
-from urllib.parse import quote_plus
+import os
+import gzip
+import json
+
+
+class geneChecker:
+    """Handles checking gene ids, retrieves symbols"""
+
+    GENE_FILES = {
+        "Homo sapiens": os.path.join(
+            os.path.dirname(os.path.realpath(__file__)),
+            "ontology_files/genes_homo_sapiens.csv.gz",
+        ),
+        "Mus musculus": os.path.join(
+            os.path.dirname(os.path.realpath(__file__)),
+            "ontology_files/genes_mus_musculus.csv.gz",
+        ),
+    }
+
+    def __init__(self, species):
```

Comment:
if species is expected to be a string add a type hint. If there are currently only 2 species supported you may want to consider making this an enum

Reply:
fixed in 4ac358decef7509c5944b70e93c9bd1e22eb45c1